staker-rewards/rewards.js

```diff
@@ -117,7 +119,17 @@ async function run() {
            )}
           `.replace(/\n/g, "\t")
       )
-    )
+
+      if (!operatorRewards.totalRewards.isZero()) {
+        rewards[
+          operatorRewards.operator
+        ] = operatorRewards.totalRewards
+          .integerValue(BigNumber.ROUND_DOWN)
```

Comment:
Do we need to `ROUND_DOWN`? Not sure about this one (cc @lukasz-zimnoch)

Reply:
If we don't `ROUND_DOWN` explicitly, the default rounding mode `ROUND_HALF_UP` is used. This doesn't crop the decimal part as expected which may cause the sum of the total rewards (calculated by the script) to be bigger than the one from the input.